modules/portal/app/views/common/footer.scala.html

```diff
@@ -57,6 +57,9 @@ <h3>@Messages("mailingList.prompt")</h3>
             <li class="@views.Helpers.maybeActive("/api")">
                 <a href="/api">API</a>
             </li>
+            <li>
```

Comment:
On second look I think you can do this more simply using the following code:

    @footerLink(controllers.portal.routes.Portal.externalPage("faq"), "faq")

The `@footerLink` function (defined at the top of the file) also adds `class="active"` to the link if that URL is the current URL, which is nice. :)